apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx

```diff
@@ -86,7 +86,13 @@ export function Accordion({
       >
         <Stack spacing={3} alignItems="center" direction="row">
           {icon}
-          <Box sx={{ maxWidth: '24ch', overflow: 'hidden' }}>
+          <Box
+            sx={{
+              maxWidth: '24ch',
+              overflow: 'hidden',
+              transform: 'translateY(-2px)'
+            }}
+          >
```

Comment:
Minor header alignment tweak in the editor accordion as per Richard's requirement

Reply:
can we use positioning instead as translateY breaks the flow of the document. 


```suggestion
          <Box
            sx={{
              maxWidth: '24ch',
              overflow: 'hidden',
              position: 'relative',
              top: 2
            }}
          >
```